services/gitdiff/gitdiff.go

```diff
@@ -1178,8 +1178,6 @@ func GetDiff(gitRepo *git.Repository, opts *DiffOptions, files ...string) (*Diff
 				} else if language, has := attrs["gitlab-language"]; has && language != "unspecified" && language != "" {
 					diffFile.Language = language
 				}
-			} else {
-				log.Error("Unexpected error: %v", err)
```

Comment:
> We should only log CheckPath errors if they are not simply due to context cancellation

Are you and I interpreting this sentence differently?
Why did you completely remove this branch then?

Reply:
The error will already be logged by the defer logger